tensorrt_llm/_torch/auto_deploy/custom_ops/cuda_backend_causal_conv.py

```
@@ -0,0 +1,293 @@
+    # INPUTS (dense but may be flattened across sequences)
+    input: torch.Tensor,  # [b, s, c_in]
+    weight: torch.Tensor,  # [c_out, c_in/groups, k] but we expect depthwise use: [c_in, k]
+    bias: Optional[torch.Tensor],
+    # METADATA
+    seq_len: torch.Tensor,  # [num_seq]
+    seq_start: torch.Tensor,  # [num_seq]
+    slot_idx: torch.Tensor,  # [num_seq]
+    # CACHES
+    conv_state_cache: torch.Tensor,  # [max_batch_size, c_in, k-1]
+    # CONSTANTS
+    stride: int,
```

Ruff (0.13.1)

Unused function argument: `stride`

(ARG001)